### PR TITLE
Choose recaptcha source to cardknox

### PR DESCRIPTION
**Pull request by pritesh-lr**
The purpose of this PR is to fix recaptcha not working in China country.
Add option **Select Recaptcha source**  to store configuration > cardknox.
Frontend google recaptcha render based on the selected source.

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/cardknox-method.js [hunk 1]
```diff
@@ -152,13 +155,23 @@ define(
                  */
                 setIfieldStyle('card-number', self.defaultStyle);
                 setIfieldStyle('cvv', self.defaultStyle);
-
+
```
Comment (mstein-ck) [inline view/frontend/web/js/view/payment/method-renderer/cardknox-method.js:158]:
```suggestion
```

File: view/frontend/web/js/recaptcha/api.js
Comment (mstein-ck) [inline view/frontend/web/js/recaptcha/api.js:1]: Where is this code from?

File: view/frontend/web/js/recaptcha/recaptcha__en.js
Comment (mstein-ck) [inline view/frontend/web/js/recaptcha/recaptcha__en.js:1]: Where is this code from?

File: view/frontend/templates/recaptcha.phtml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+/** @var $viewModel \CardknoxDevelopment\Cardknox\ViewModel\Data */
+$viewModel = $block->getViewModel();
+$isEnabledReCaptcha = $viewModel->isEnabledReCaptcha();
+$selectReCaptchaSource = $viewModel->getSelectReCaptchaSource();
+
+
+if ($isEnabledReCaptcha && $selectReCaptchaSource == "recaptcha.net") : ?>
+    <script>
+        require(['https://www.recaptcha.net/recaptcha/api.js']);
```
Comment (mstein-ck) [inline view/frontend/templates/recaptcha.phtml:11]: Is this the same file as the js below?

**Comment by mstein-ck** [comment]
Can you resolve the code smells?

File: view/frontend/templates/recaptcha.phtml (hunk 2 above)
Comment (pritesh-lr) [reply view/frontend/templates/recaptcha.phtml:11]: @mstein-ck, I updated the code and removed both recaptcha__en.js & api.js

File: view/frontend/web/js/recaptcha/api.js
Comment (pritesh-lr) [reply view/frontend/web/js/recaptcha/api.js:1]: @mstein-ck , removed js now.

File: view/frontend/web/js/recaptcha/recaptcha__en.js
Comment (pritesh-lr) [reply view/frontend/web/js/recaptcha/recaptcha__en.js:1]: @mstein-ck , removed js now

**Comment by pritesh-lr** [comment]
> 

Please review it again. I removed unnecessary codes.

**Comment by pritesh-lr** [comment]
> Can you resolve the code smells?


Please review it again. I removed unnecessary codes.

**Review by mstein-ck — APPROVED** [review]

**Comment by pritesh-lr** [comment]
@mstein-ck, Can you please re-run the job for pass the auto run PHPUnit test?

1. Go to this URL: https://github.com/cardknox/magento2_cardknox/actions/runs/5210493531/job/14103574035
2. Click on mentioned icon, see screenshot: https://prnt.sc/aPZIeOCRl02v
3. Click on `Re-run jobs` button, see screenshot: https://prnt.sc/ef-ToYFqNIYZ
4. It will pass auto run `PHPUnit` test

Thanks

**Pull request merged**